<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Maditor::View::MainWindow</class>
 <widget class="QMainWindow" name="Maditor::View::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maditor</string>
  </property>
  <property name="iconSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="Maditor::View::CentralWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="Maditor::View::DocumentTabWidget" name="tabWidget">
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionLoadProject"/>
    <addaction name="actionSettings"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDocument">
    <property name="title">
     <string>Document</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDocument"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="modulesDockWidget">
   <property name="windowTitle">
    <string>Modules</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="modulesDockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QTableView" name="modulesWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="projectDockWidget">
   <property name="windowTitle">
    <string>Project</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="projectDockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_5">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="Maditor::View::TreeView" name="projectWidget">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="mediaDockWidget">
   <property name="windowTitle">
    <string>Media</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="mediaDockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_6">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="Maditor::View::FilesView" name="mediaWidget">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="logsDockWidget">
   <property name="windowTitle">
    <string>Logs</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="logsDockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_7">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="Maditor::View::LogWidget" name="logsWidget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="appStatsDockWidget">
   <property name="windowTitle">
    <string>Application Statistics</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="appStatsDockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_10">
     <item row="1" column="0">
      <widget class="Maditor::View::TreeView" name="performanceWidget">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="Maditor::View::AppStatsWidget" name="appStatsWidget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="inspectorDockWidget">
   <property name="windowTitle">
    <string>new Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="inspectorDockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="Maditor::View::InspectorView" name="inspectorWidget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNewProject">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionLoadProject">
   <property name="text">
    <string>Load Project</string>
   </property>
  </action>
  <action name="actionSetup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Setup</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/start.jpg</normaloff>:/icons/icons/start.jpg</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/pause.png</normaloff>:/icons/icons/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/Stop.jpg</normaloff>:/icons/icons/Stop.jpg</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionSetup_No_Debug">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Setup (No Debug)</string>
   </property>
  </action>
  <action name="actionShutdown">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Shutdown</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionKill">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Kill</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Maditor::View::TreeView</class>
   <extends>QTreeView</extends>
   <header>treeview.h</header>
  </customwidget>
  <customwidget>
   <class>Maditor::View::FilesView</class>
   <extends>QTreeView</extends>
   <header>project/filesview.h</header>
  </customwidget>
  <customwidget>
   <class>Maditor::View::AppStatsWidget</class>
   <extends>QWidget</extends>
   <header>application/appstatswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Maditor::View::LogWidget</class>
   <extends>QWidget</extends>
   <header>logs/logwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Maditor::View::InspectorView</class>
   <extends>QWidget</extends>
   <header>application/inspector/inspectorview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Maditor::View::DocumentTabWidget</class>
   <extends>QTabWidget</extends>
   <header>documents/documenttabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Maditor::View::CentralWidget</class>
   <extends>QWidget</extends>
   <header>documents/centralwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
